drivers/CoreLED/include/CoreLED.h

```diff
@@ -49,9 +58,22 @@ class CoreLED : public interface::LED
 		}
 
 		std::fill(_colors.begin() + start, _colors.begin() + end + 1, color);
+		std::fill(_brightness.begin() + start, _brightness.begin() + end + 1, default_brightness);
+	}
+
+	void fadeToBlackBy(const uint8_t &value) override
+	{
+		for (auto &b: _brightness) {
+			auto new_brightness = b - value;
+			if (new_brightness > minimum_brightness) {
+				b = new_brightness;
+			} else {
+				b = minimum_brightness;
+			}
+		}
```

Comment:
on n'appelle pas `show`?

```suggestion
	void fadeToBlackBy(const uint8_t &value) override
	{
		for (auto &b: _brightness) {
			if (b == minimum_brightness) {
				continue;
			}
			b = b - value;
		}
```

Reply:
@HPezz tu as pas fait cette modification du if pour simplifier